cfgov/jinja2/v1/_includes/organisms/email-popup/base.html

```diff
@@ -1,60 +1,77 @@
     <div class="o-email-popup_header u-clearfix">
         <div class="close">
             <button class="a-btn a-btn__link">Close {{ svg_icon('delete') }}</button>
         </div>
     </div>
+
     <div class="o-email-popup_body"
          style="background-image: url('{{ static(popup_image) }}')">
-        <div class="o-email-signup">
-            <h2>{{ popup_headline }}</h2>
-            <form id="{{ ('o-email-signup_' ~ range(1, 100) | random) }}"
-                  class="o-form o-form__email-signup u-clearfix"
-                  action="{{ url('govdelivery') }}"
-                  method="POST"
-                  enctype="application/x-www-form-urlencoded"
-                  novalidate>
-                <p>{{ popup_content }}</p>
-                <div class="m-form-field-with-button" data-qa-hook="formfield-with-button">
-                    <div class="form-group">
-                        <input id="form_18"
-                               type="email"
-                               placeholder="Enter your email address"
-                               name="email"
-                               class="m-form-field-with-button_field a-text-input"
-                               required>
-                    </div>
-                    <p>
-                        <a href="{{ privacy_link_url }}"
-                           target="_blank"
-                           rel="noopener noreferrer">Privacy Act statement</a>
-                        <br/>
-                    </p>
-                    <input class="a-btn a-btn__full-on-xs" type="submit" value="Sign up">
-                </div>
-                <div class="form-group">
-                    <input type="hidden" name="code" value="{{ mailing_list_code }}">
-                </div>
-            </form>
-            <div class="o-email-signup_footer">
-                {% import 'molecules/notification.html' as notification with context %}
-                {{ notification.render('default', false, '') }}
+        <h2>{{ popup_headline }}</h2>
+        <p>{{ popup_content }}</p>
+        <form class="o-form o-form__email-signup u-clearfix"
+              id="{{ 'o-email-signup_' ~ form_id }}"
+              method="POST"
+              action="{{ url('govdelivery') }}"
+              enctype="application/x-www-form-urlencoded"
+              novalidate>
+            <div class="m-form-field">
+                <label class="u-visually-hidden" for="{{ 'email_' ~ form_id }}">
+                    Email address
+                </label>
+                <input class="a-text-input a-text-input__full"
+                       id="{{ 'email_' ~ form_id }}"
+                       name="email"
+                       type="email"
+                       placeholder="Enter your email address"
+                       required>
             </div>
-        </div>
+
+            <div class="m-btn-group">
+                <input class="a-btn a-btn__full-on-xs" type="submit" value="Sign up">
```

Comment:
Use `<button>` instead of `type="submit"`?

Reply:
Yeah, I think that makes sense.

Reply:
Updated in https://github.com/cfpb/cfgov-refresh/pull/4838/commits/75324dfce4ced3b5116c004b5922521ac15ddf4e